src/modules/charging-import/routes.js

```diff
@@ -7,10 +7,5 @@ module.exports = [
     method: 'post',
     handler: controller.postImportChargingData,
     path: '/import/1.0/charging'
-  },
-  {
```

Comment:
You sure you want to remove the endpoint? I think it could be handy for manually triggering the import. Ignore this comment if you're confident/happy that this is no longer required.

Reply:
The job that this endpoint triggered is no longer in the code.
It may be handy to add an endpoint that triggers the NALD import, as that is what now populates the charge version metadata, and at the moment that is only done on a cron schedule.